templates/maas/index.html

```diff
@@ -1,1060 +1,700 @@
-             alt="Deploy Ubuntu, Redhat, CentOS, Windows on IBM, HP, Lenovo, Quanta Cloud Technology, Dell, Open Compute Project, Huawei, Cisco" />
-      </div>
-    </div>
-    <div class="row">
-      <ul class="p-list row">
-        <li class="p-list__item is-ticked is-large col-4 p-heading--5 u-no-margin--top">
-          <a href="https://ubuntu.com/certified?category=Server">Works on all Canonical certified servers</a>
-        </li>
-        <li class="p-list__item is-ticked is-large col-4 p-heading--5 u-no-margin--top">
-          Discovers servers in racks, chassis and data centre networks
-        </li>
-        <li class="p-list__item is-ticked is-large col-4 p-heading--5 u-no-margin--top">
-          Supports major system BMCs and chassis controllers
-        </li>
-      </ul>
-    </div>
-  </section>
+{% block content %}
+  {% call(slot) vf_hero(
+    title_text='Bare metal and data center automation made easy',
+    layout='50/50',
+    is_split_on_medium=true
+    ) -%}
+    {%- if slot == 'description' -%}
+      <h2 class="p-text--default">Turn hardware into programmable, cloud-like bare metal infrastructure</h2>
+    {%- endif -%}
+    {%- if slot == 'cta' -%}
+      <a href="/docs/maas-in-thirty-minutes" class="p-button--positive">Try MAAS</a>
+      <a href="/maas/contact-us" class="p-button">Get in touch</a>
+    {%- endif -%}
+    {%- if slot == 'image' -%}
+      <div class="u-embedded-media">
+        <iframe class="u-embedded-media__element"
+                width="441"
+                height="247"
+                src="https://www.youtube.com/embed/eHfaiKlUYsM?showinfo=0&controls=2&modestbranding=1"
+                title="Watch the MAAS video on YouTube"
+                frameborder="0"
+                allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture;"
+                allowfullscreen>
+        </iframe>
+      </div>
+    {%- endif -%}
+  {% endcall -%}
+
+  {{ vf_basic_section(title={"text": "Why automate your data center with MAAS"},
+    is_split_on_medium=true,
+    items=[
+    {
+    "type": "description",
+    "padding": "shallow",
+    "item": {
+    "type": "html",
+    "content": '
+    <p>MAAS is an open source platform that provides a centralized environment for managing and provisioning physical servers as if they were cloud resources.</p>
+    <p>By turning complex tasks into automated workflows, MAAS helps teams move faster and focus on innovation instead of infrastructure.</p>
+    <h3 class="p-heading--5 u-no-padding u-no-margin">Lower costs with automation</h3>
+    <p>Reduce costs and human error by automating the entire lifecycle of your machines, from initial setup to final decommissioning.</p>
+    <h3 class="p-heading--5 u-no-padding u-no-margin">Scale your hardware resources effectively</h3>
+    <p>Treat your physical servers as a flexible, on-demand resource pool that can be provisioned and scaled instantly.</p>
+    '
+    }
+    },
+    {
```

Comment:
All these macros could do with some proper indentation. Maybe the formatter has done something strange to them, so they might need some manual fixes.

Reply:
Yep, djlint formatter is completly messing up with those jinja macro calls with deeply nested values.

I'll bring it up to discussion maybe on MM channel.